src/main/java/Jelly/main/Jelly.java

```diff
@@ -24,11 +32,21 @@ private Jelly(String filePath) throws JellyException {
             System.out.println(e.getMessage());
         }
     }
+
+    /**
+     * The main initialiser for the Jelly Chat bot.
+     *
+     * @param args
```

Comment:
I do not think  "@param args" help much
And I believe that you do not need to document every method as well.